lib/vdi.py

```diff
@@ -89,3 +93,9 @@ def param_clear(self, param_name):
     def param_remove(self, param_name, key, accept_unknown_key=False):
         _param_remove(self.sr.pool.master, self.xe_prefix, self.uuid,
                       param_name, key, accept_unknown_key)
+
+    def wait_for_coalesce(self):
+        # It is necessary to wait a long time because the GC can be paused for more than 5 minutes.
```

Comment:
does it depends on size ?  can the value be computed ? btw shannon would double the value and set to 10min